src/Magnum/ImGuiIntegration/Widgets.h

```diff
@@ -70,15 +104,33 @@ inline void image(GL::Texture2D& texture, const Vector2& size,
     padding
 @param backgroundColor  Background color, default @cpp 0x00000000_rgbaf @ce
 @param tintColor        Tint color, default @cpp 0xffffffff_rgbaf @ce
```

Comment:
I'd remove the `@param` lines here, as it's enough to have them in the non-deprecated API.